src/tablo/tabloVariables.scss

```diff
@@ -3,12 +3,38 @@
 $border-color: $brc-silver !default;
 $border-width: 1px !default;
 
+$table-border-color: $border-color !default;
+$table-border-width: $border-width !default;
+$table-border-left-color: $table-border-color !default;
```

Comment:
Well, I imagined it could be useful to remove some border, but I find very hard we want to have different colors for each of them. Maybe we could keep only 1 var for color and 4 vars for widths?